packages/tabs/src/index.tsx

```diff
@@ -279,14 +296,20 @@ export const TabList = forwardRefWithAs<TabListProps, "div">(function TabList(
 
   let handleKeyDown = wrapEvent(
     function(event: React.KeyboardEvent) {
-      if (event.key === "ArrowDown") {
+      if (
+        orientation === "vertical"
+          ? isRTL
+            ? event.key === "ArrowLeft"
+            : event.key === "ArrowRight"
```

Comment:
One thing I'm just realizing: this doesn't take into account the relative order of TabsList and TabPanels. For instance:

```jsx
<Tabs orientation="vertical">
  <TabPanels>
    <TabPanel>
      <h1>one!</h1>
      <p>Here's some example content.</p>
      <button>yo</button>
    </TabPanel>
    <TabPanel>
      <h1>two!</h1>
      <p>Here's some example content.</p>
    </TabPanel>
    <TabPanel>
      <h1>three!</h1>
      <p>Here's some example content.</p>
    </TabPanel>
  </TabPanels>

  <TabList>
    <Tab>One</Tab>
    <Tab>Two</Tab>
    <Tab>Three</Tab>
  </TabList>
</Tabs>
```

Assuming we're in LTR mode, TabPanels will show up on the left (which is correct), but the left arrow key won't focus the panel.

I'll think about how to address that, but let me know if you all have any ideas. Could actually be a good reason to make the options more explicit as @chancestrickland mentioned in his TODO i.e. `horizontal-top`, `horizontal-bottom`, etc.

Reply:
I took a shot at detecting the relative order (while still taking isRTL into account) in https://github.com/reach/reach-ui/pull/486/commits/f07be1d61eb10ce2c718cba6d0cd2704efb563a4.